proto/encryptionpb.proto

```diff
@@ -11,6 +11,16 @@ option (rustproto.lite_runtime_all) = true;
 
 option java_package = "org.tikv.kvproto";
 
+// General encryption metadata for any data type.
+message EncryptionMeta {
+    // Whether data is encrypted.
+    bool is_encrypted = 1;
```

Comment:
`EncryptionMeta` is not `nil`, but `is_encrypted` is `false`. What is the corresponding situation?

Reply:
The two case should both treated as the region boundary keys are not encrypted.